deploy/overlays/dev/kustomization.yaml

```diff
@@ -34,6 +34,9 @@ configMapGenerator:
     literals:
       - mixerProject=datcom-website-dev
       - serviceName=website-esp.endpoints.datcom-website-dev.cloud.goog
+  - name: ai-config
+    files:
+      - ai.yaml
```

Comment:
can move this to base/kustomization.yaml as well and define empty ai.yaml for all env that has no model deployed

Reply:
Resolved offline.